pkg/cmd/cli/cmd/status.go

```diff
@@ -160,7 +160,9 @@ func (o StatusOptions) RunStatus() error {
 		if err != nil {
 			return err
 		}
-		data, err := dot.Marshal(g, o.namespace, "", "  ", false)
+		// DOT graph names cannot contain dashes unless it is a double-quoted string
+		// See http://www.graphviz.org/doc/info/lang.html
+		data, err := dot.Marshal(g, fmt.Sprintf("%q", o.namespace), "", "  ", false)
```

Comment:
Is it possible for `StatusOptions.namespace` to contain anything other than lower-case letters, numbers, and dashes (since it corresponds with project name)?  If that is all it contains then `quote` would just use the `%q` string format?